packages/models/proto/v2/eservice/events.proto

```diff
@@ -103,3 +103,8 @@ message EServiceRiskAnalysisAddedV2 {
   string riskAnalysisId = 1;
   EServiceV2 eservice = 2;
 }
+
+message EServiceRiskAnalysisDeletedV2 {
```

Comment:
The event should also be added in the `V1` events, to support previous events - see for example what was done in: https://github.com/pagopa/interop-be-monorepo/pull/231/files#diff-30a4474e2b924ad118ae9df15cf84bc3a0a663cae0ca933b4d6d1de43bfdf531R64

This means also adding it in the `EServiceEventV1` in `eserviceEvents.ts`, handling it in the readmodel-writer `consumerServiceV1.ts` etc etc

Reply:
Updated